TransIT.DAL/Models/Entities/Currency.cs

```diff
@@ -23,5 +24,6 @@ public Currency()
         public virtual User Mod { get; set; }
         
         public virtual ICollection<Supplier> Supplier { get; set; }
+        public virtual ICollection<PartIn> PartInNavigation { get; set; }
```

Comment:
you have different names of PartsInNavigation (int Part.cs with s) and in here without), check all names for this property should be the same, - PartsInNavigation